test/IECoreHoudini/FromHoudiniPolygonsConverter.py

```diff
@@ -834,5 +834,39 @@ def testRename( self ) :
 		self.assertTrue( isinstance( converter, IECoreHoudini.FromHoudiniPolygonsConverter ) )
 		self.assertTrue( isinstance( converter.convert(), IECoreScene.MeshPrimitive ) )
 
+
+	def testRenamedUVsExportedWithInterpretation( self ):
+
+		box = self.createBox() # create cube geometry
+		quickShade = box.createOutputNode("uvquickshade") # quickly generate orthographically projected UVs
+		attribute = quickShade.createOutputNode("attribute") # rename the 'uv' attribute 'foo'
+		attribute.parm("frompt0").set("uv")
+		attribute.parm("topt0").set("foo")
+
+		convertToCortex = attribute.createOutputNode("ieCortexConverter")
+		convertToCortex.parm("resultType").set(0) # convert to cortex
```

Comment:
use the enum please

Reply:
So the enum is in another test https://github.com/ImageEngine/cortex/blob/92d3dd7c275da331f3cd2f3467189f0b621edda6/test/IECoreHoudini/CortexConverterSop.py#L47

At the time you mentioned we should put this somewhere else. Any suggestions?

Reply:
gah, I guess its not the same value set as `IECoreHoudini.SceneCacheNode.GeometryType` is it... ok, nevermind then...